build.assets/Dockerfile

```
@@ -83,6 +84,22 @@ RUN (groupadd ci --gid=$GID -o && useradd ci --uid=$UID --gid=$GID --create-home
 RUN (curl -L https://github.com/coreos/etcd/releases/download/v3.3.9/etcd-v3.3.9-linux-${BUILDARCH}.tar.gz | tar -xz && \
      cp etcd-v3.3.9-linux-${BUILDARCH}/etcd* /bin/)
 
+# Install libcbor.
+RUN git clone --depth=1 https://github.com/PJK/libcbor.git -b v0.9.0 && \
+  cd libcbor && \
+  cmake -DCMAKE_BUILD_TYPE=Release -DCBOR_CUSTOM_ALLOC=ON . && \
+  make && \
+  make install
+
+# Install libfido2.
+# Depends on libcbor, libssl-dev, zlib and libudev-dev.
+RUN git clone --depth=1 https://github.com/Yubico/libfido2.git -b 1.10.0 && \
+  cd libfido2 && \
+  cmake -DCMAKE_BUILD_TYPE=Release . && \
+  make && \
+  make install
```

Also applies to libcbor
```suggestion
  cmake --build . --config Release && \
  cmake --install . --config Release
```